### PR TITLE
Action Recognition models update

### PR DESCRIPTION
**Pull request by evgeny-izutov**
* Renames "Action Recognition 2" to "Action Recognition" problem.
* Adds tests for the Action Recognition problem.
* Adds support of args.classes.
* Adds custom action recognition model.

### CODE REVIEW

File: models/action_recognition/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Action Recognition
+
+## Pre-trained models
+
+This repo contains scripts and tutorials for action recognition models training.
+
+* [Custom Action Recognition](model_templates/custom-recognition/README.md) - models that are
+  trained on the Kinetics-700 dataset and suitable for transfer learning.
+* [Gesture Recognition](model_templates/gesture-recognition/README.md) - models that are able
+  to recognize hand gestures (e.g. American Sign Language (ASL) and Jester).
```
Comment (Ilya-Krylov) [inline models/action_recognition/README.md:10]: putting link to Jester would be great

File: problems.yaml [hunk 2]
```diff
@@ -58,14 +58,20 @@ domains:
...
-#  problems:
-#  - title: Gesture Recognition
-#    description:  >-
-#      Gesture recognition is a computer technology being used in a variety of applications
-#      that carry outs video stream classification.
-#    subtitle: Recognizing gestures in live video streams
-#    cvat_schema: '[{}]'
+- title: Action Recognition
+  problems:
+  - title: Custom Recognition
```
Comment (Ilya-Krylov) [inline problems.yaml:63]: Custom Action Recognition will be clearer

File: models/action_recognition/model_templates/custom-recognition/s3d-rgb-mobilenet-v3/template.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+name: s3d-rgb-mobilenet-v3
+domain: Action Recognition
+problem: Custom Recognition
```
Comment (Ilya-Krylov) [inline models/action_recognition/model_templates/custom-recognition/s3d-rgb-mobilenet-v3/template.yaml:3]: Custom Action Recognition will be clearer

File: README.md [hunk 4]
```diff
@@ -40,8 +40,9 @@ inference.
 After installation, you are ready to train your own models, evaluate and use
 them for prediction.
 
-* [Action Recognition](models/action_recognition_2)
-  - [Gesture Recognition](models/action_recognition_2/model_templates/gesture-recognition)
+* [Action Recognition](models/action_recognition)
+  - [Custom Action Recognition](models/action_recognition/model_templates/custom-recognition)
```
Comment (Ilya-Krylov) [inline README.md:44]: custom-action-recognition will be clearer

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

File: models/action_recognition/README.md (hunk 1 above)
Comment (evgeny-izutov) [reply models/action_recognition/README.md]: done

File: models/action_recognition/model_templates/custom-recognition/s3d-rgb-mobilenet-v3/template.yaml (hunk 3 above)
Comment (evgeny-izutov) [reply models/action_recognition/model_templates/custom-recognition/s3d-rgb-mobilenet-v3/template.yaml]: done

File: README.md (hunk 4 above)
Comment (evgeny-izutov) [reply README.md]: done

**Review by Ilya-Krylov — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
@evgeny-izutov merge develop branch into yours and fix your tests.

**Comment by evgeny-izutov** [comment]
@Ilya-Krylov I've temporally disabled tests for the custom-action-recignition problem due to significant problems with config mismatching between pre-train and template scenarios. In future it will require some ideology changes to support tests for the "custom" templates. Currently I would prefer merge such custom template without tests fortunately it reuses the same code base as gesture-recognition templates.

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**